### DISCUSSION TITLE
Source code for backend of docs.mojolicious.org and mojojs.org/docs

### DISCUSSION THREAD

**Discussion by andrew-grechkin** [mojolicious/mojo#2100] [Q&A] [answered]
I have tried to find source code for backend/frontend of https://docs.mojolicious.org and https://mojojs.org/docs but unfortunately was unable to find any.

Even though I see some assets used on these pages in mojo.git repo I don't see the code where the actual work of converting POD and md to html is happening and template for the frontent pages as well.

I have tried to google and checked cpan and github repos related to mojo.
Can you please point me to the repo(s) if that is open source
Thanks!

**Comment by Grinnz** [comment, marked as answer]
https://github.com/mojolicious/mojolicious.org/blob/main/lib/Mojolicious/Plugin/MojoDocs.pm
